packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -81,6 +81,22 @@ export class WorkspaceFrontendContribution implements CommandContribution, MenuC
         });
     }
 
+    registerKeybindings(keybindings: KeybindingRegistry): void {
```

Comment:
`implements KeybindingContribution` is missing

Reply:
done thanks!